src/style.css

```diff
@@ -1,3 +1,60 @@
 body {
+  font-family: Roboto, "Times New Roman", sans-serif;
+  margin: 0 50px;
+  --link-color: #00acdc;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+}
+
+.header__logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
   margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.nav__list li:not(:first-child) {
```

Comment:
[Don't use tag names for styling (except `html` and `body`)](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#css)